pkg/api/register.go

```diff
@@ -51,10 +55,27 @@ func Resource(resource string) unversioned.GroupResource {
 	return SchemeGroupVersion.WithResource(resource).GroupResource()
 }
 
-func AddToScheme(scheme *runtime.Scheme) {
-	if err := Scheme.AddIgnoredConversionType(&unversioned.TypeMeta{}, &unversioned.TypeMeta{}); err != nil {
+var (
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes, addDefaultingFuncs)
+	AddToScheme   = SchemeBuilder.AddToScheme
+)
+
+func init() {
+	// TODO(lavalamp): move this call to scheme builder above.
+	// Can't fix this without doing proper dependency tracking-- otherwise
+	// this will get double registered. (Consequence of not fixing this is
+	// that it only ever gets registered in the main api.Scheme, even
+	// though everyone needs these.)
```

Comment:
It works, but it's terrible design to have a global object that can't be reproduced hermetically for testing.